server/services/database/applicantProfile.js

```diff
@@ -2,6 +2,47 @@ const { Pool } = require("pg");
 const config = require("../../config");
 const pool = new Pool(config);
 
+<<<<<<< HEAD
```

Comment:
the merge conflict is not fixed  .. this line `<<<HEAD` should not be there. Please update the PR again, and make sure it works locally after the update (this will break the code, and it shouldn't work)

Reply:
you need to make a choice between what's in `<<< HEAD` and what's in your branch